container/kvm/cache_internal_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package kvm
```

Comment:
1. Please add a line break between copyright block and package declaration.
2. Should this be a "kvm" package or "kvm_test" package?